frameworks/solid/src/primitives/useField/useField.test.tsx

```
@@ -405,5 +409,67 @@ describe('useField', () => {
+    test('should not duplicate element registration after an array reorder', async () => {
+      const schema = v.object({
+        todos: v.array(v.object({ label: v.string() })),
+      });
+      let formStore: FormStore<typeof schema> | undefined;
+
+      function Row(props: {
+        form: FormStore<typeof schema>;
+        index: number;
+      }): JSX.Element {
+        const field = useField(props.form, {
+          // eslint-disable-next-line solid/reactivity
```

ESLint

[error] Definition for rule 'solid/reactivity' was not found.

(solid/reactivity)